rodrigoprimo/woocommerce
# Use the latest PHP to run Travis jobs for PHPCS, e2e and code coverage

### Changes proposed in this Pull Request:

This PR simply changes Travis configuration to use the latest stable PHP version to run jobs for PHPCS, e2e tests, and unit tests code coverage. Since we run those checks against a single PHP version, it is better to use the latest.

### How to test the changes in this Pull Request:

1. Check the PHP version used to run the three Travis jobs mentioned above.
2. Make sure the other Travis jobs are working properly.
